UnityProject/Assets/Scripts/Items/ClothingItem.cs

```diff
@@ -93,6 +93,8 @@ public virtual void SetReference(GameObject Item)
 		UpdateReferenceOffset();
 		if (Item == null)
 		{
+			GameObjectReference = null; // Remove the item from equipment
```

Comment:
this causes an nre when you remove your clothing after spawning:

```
var unequippedClothing = GameObjectReference.GetComponent<ClothingV2>();
```

Need to ensure this hook is still fired before nulling out GameObjectReference.

Reply:
Was the NRE in client or server?

Reply:
Both. I hosted in editor, joined as client, then as client removed my suit and dropped it on the ground.

Reply:
Looking at it.  Thanks for finding it.

Reply:
Corrected by moving the null assignation bellow where the reference is needed.